cmd/flow-dps-executor/README.md

```diff
@@ -0,0 +1,173 @@
+# Flow DPS Executor
+
+## Description
+
+The Flow DPS Executor provides access to Flow DPS Server's index through a REST API.
+It can be used to execute Cadence scripts at an arbitrary block height of a fork.
+It uses the Flow DPS Server's GRPC API as the backend to query the required data.
+
+## Running the Executable
+
+```sh
+Usage of flow-dps-executor:
+  -a, --api string     host for GRPC API server
+  -l, --level string   log output level (default "info")
+  -p, --port uint16    port to host Executor API on (default 8080)
+```
+
+## Usage over HTTP
+
+The Flow DPS Executor accepts POST requests at the `/execute` endpoint.
+Example of a correct HTTP request payload sent to the Executor is shown below:
+
+```json
+{
+    "height": 10,
+    "script": "script text",
+    "arguments": [
+        "type1(value1)",
+        "type2(value2)",
+        "type3(value3)" 
+    ]
+}
+```
+
+If the provided script expects no arguments, the `arguments` string array can be ommitted.
+Example of an HTTP response payload sent by the Executor after successful script execution is shown below:
+
+```json
+{
+    "height": 10,
+    "script": "script text",
+    "arguments": [
+        "type1(value1)",
+        "type2(value2)",
+        "type3(value3)" 
+    ],
+    "result": <script result>
```

Comment:
Having this in a JSON snippet will break formatting. WDYT of having it as a string instead?

```suggestion
    "result": "script result"
```